### DISCUSSION TITLE
Dose level in CST

### DISCUSSION THREAD

**Discussion by Mariam950-wp** [e0404/matRad#777] [General]
Dear Friends. 

In cst struct of TG119 phantom of matrad test data, there are some treatment planing parameters defined, one of which is a level of dose. I wonder how the level dose is related to an actual dose in Gray. So how it is defines that target should get a specific dose?

**Comment by wahln** [comment]
The Standard objectives are of type "squared overdosing" and "squared deviation". 

The reference level for squared deviation is the total dose (summed over all fractions) and means that the optimizer will minimze the least squares to this parameter. 

For squared overdosing, the parameter is a lower limit, in the sense that the square of _positive_ difference will be minimized.

For somewhat historical reasons we stick with the convention that the prescriptions are given as total dose, while the dose that is by default displayed in matRad is the fraction dose. This is sometimes confusing, but when setting "60" as a dose parameter in the target and setting number of fractions to "30", you will consequently see target doses of 2 Gy / fraction in matRad.

**Comment by Mariam950-wp** [comment]
Thanks for your answer 🙂

and I have n emore question. I know that matrad is based on interpolations and it is not monte-carlo simulation but how can I guess how many protons were simulated in a fraction if it is possible. I see defauls number of particles for a ray in documentation, but not actual one.

Thanks
________________________________

**Reply by wahln** [reply]
Our dose kernels in the particle machine data is normalized to 1e6 primaries / weight. That means a value of 1 in the weight vector after optimization is 1e6 particles (for a fraction).
This means `nParticles = 1e6 * sum(resultGUI.w)`
